openbas-front/src/admin/components/simulations/simulation/validation/expectations/ManualExpectationsValidationForm.tsx

```diff
@@ -114,15 +118,45 @@ const ManualExpectationsValidationForm: FunctionComponent<FormProps> = ({ expect
                          />)}
         {withSummary && (<Typography variant="h3">{expectation.inject_expectation_user ? t('Player') : t('Team')}</Typography>)}
         {withSummary && targetLabel(expectation)}
-        <MuiTextField
-          className={withSummary ? classes.marginTop_2 : classes.scoreAcc}
-          variant="standard"
-          fullWidth
-          label={t('Score')}
-          type="number"
-          error={!!errors.expectation_score}
-          helperText={errors.expectation_score && errors.expectation_score?.message ? errors.expectation_score?.message : `${t('Expected score:')} ${expectation.inject_expectation_expected_score}`}
-          inputProps={register('expectation_score')}
+        <Grid container={true} spacing={3}>
+          <Grid item={true} xs={6}>
+            <MuiTextField
+              className={withSummary ? classes.marginTop_2 : classes.scoreAcc}
+              variant="standard"
+              fullWidth
+              label={t('Score')}
+              type="number"
+              error={!!errors.expectation_score}
+              helperText={errors.expectation_score && errors.expectation_score?.message ? errors.expectation_score?.message : `${t('Expected score:')} ${expectation.inject_expectation_expected_score}`}
+              {...register('expectation_score')}
+              InputProps={{
+                inputProps: {
+                  min: 0,
+                  max: 100,
+                },
+              }}
+            />
+          </Grid>
+          <Grid item={true} xs={6}>
+            <Select
+              fullWidth
+              value={watch('expectation_score') < expectation.inject_expectation_expected_score ? 'Failed' : 'Success'}
+              onChange={(event) => setValue('expectation_score', event.target.value === 'Success' ? 100 : 0)}
+              renderValue={(value) => {
+                return value;
+              }}
+              sx={{ marginTop: selectPaddingTop }}
```

Comment:
It's better to wrap the component with padding rather than give a new property to it